docs/AboutUs.md

```diff
@@ -38,15 +38,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
-### Jean Doe
+### Jonah Tan
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jonahtan.jpg" width="200px">
```

Comment:
I suggest changing this to .png according to the specifications. Similarly, change to file itself to .png too.